SDK - Document color theming limitation (#54351)
metabase/metabase: docs/embedding/sdk/appearance.md

  <StaticDashboard dashboardId={1} />
</MetabaseProvider>
```

> ### Limitations
> 
> Colors configured in the visualization settings of a question will take precendence over those configured in the theme.
> 
> 
> 
> 
> 
> 

Reviewer: ```suggestion
## Limitations

Colors configured in a question's visualization settings will override theme colors.
```
